go/kbfs/libkbfs/folder_branch_ops.go

```diff
@@ -8852,6 +8852,10 @@ func (fbo *folderBranchOps) SetSyncConfig(
 		fbo.log.CDebugf(ctx, "Starting full deep sync")
 		_ = fbo.kickOffRootBlockFetch(ctx, md)
 	}
+
+	fbo.config.Reporter().Notify(ctx, syncConfigChangeNotification(
```

Comment:
Nothing above actually checks and returns early if the new config is the same as the old config, so this could send unnecessary notifications.  I don't care much either way, so if that matters for the GUI we should maybe add a check like that above.

Reply:
If the new config is identical to the old one should we bail out at top? Setting the config also bails out prefetches, potentially triggers mark-and-sweep and starts new prefetching, even if the config is identical to the old one.

Reply:
I ended up just rebasing and adding the check to the notify call.